packages/lesmis-server/src/routes/ReportRoute.ts

```diff
@@ -58,10 +59,10 @@ export class ReportRoute extends Route {
           {
             // Report server can accept arrays so the parameters are plural
             organisationUnitCodes: entityCode,
-            projectCodes: LESMIS_PROJECT_NAME,
+            hierarchy: LESMIS_HIERARCHY_NAME,
             ...this.req.query,
           },
-          this.req.body,
+          {},
```

Comment:
Body is no longer required, but it will be removed as part of a different PR: https://github.com/beyondessential/tupaia/pull/2866